BUILD_WSL.md

```diff
@@ -0,0 +1,31 @@
+# Compiling hashcat for Windows with Windows Subsystem for Linux.
+
+Tested on a Windows 10 (x64 build 18363), should also work to build hashcat for Windows on Linux.
+
+### Installation ###
+
+Enable WSL.
+
+Run bash (win+r bash) and make sure to install additional dependencies necessary for hashcat compilation
+```
+sudo apt install gcc-mingw-w64-x86-64 make git
+git clone https://github.com/hashcat/hashcat
+git clone https://github.com/win-iconv/win-iconv 
+cp hashcat/tools/win-iconv-64.diff win-iconv/
```

Comment:
why copy this diff file without applying it with the "patch" command ? this step makes no sense to just copy a diff file... There is something missing here like "patch < win-iconv-64.diff" etc... but this needs to be tested. The way it's explained here to just copy a file makes no sense

Reply:
good spot, copy pasted wrong probably